### DISCUSSION TITLE
Unable to push Styles & Scripts

### DISCUSSION THREAD

**Discussion by abbasmashaddy72** [wire-elements/modal#311] [Q&A] [answered]
I had Created a CK Editor Component

It works perfectly fine when I use it in my single page view

But Not working when I use it in my Modal View

When I did a check, I found out that Custom Styles & Scripts were not getting pushed
 
Please Help me Out

**Comment by abbasmashaddy72** [comment]
@PhiloNL @thermiteplasma 

Anyone, Please Help Me Out.

**Comment by PhiloNL** [comment, marked as answer]
@abbasmashaddy72 You can't push CSS or JS, as Livewire does not support this.
If you want to initialize JS inside a modal you can use Alpine:

```html
<!-- This will execute the JS when the modal is initialized -->
<div x-init="initCkEditor()"></div>
```

**Reply by abbasmashaddy72** [reply]
Thanks Your Answer Worked

Need to make Some Change If Still Having Issues I will Revert Back

**Reply by odin88** [reply]
I have the same issues .. I am using QuillJS and create a livewire component call Quill and include as a form in one of the modal but somehow it can only run the init in the modal itself and cannot invoke in another livewire model 1 level deep.

**Reply by abbasmashaddy72** [reply]
Can show the modal view code where the issue is

@odin88 I will try my best to help you
